src/common/types/featureSupportLevel.ts

```diff
@@ -4,7 +4,7 @@ import { FeatureSupportLevel } from '../api/types';
 type ArrayElementType<TWhere> = TWhere extends (infer U)[] ? (U extends object ? U : never) : never;
 
 type Features = Required<ArrayElementType<FeatureSupportLevel['features']>>;
-export type FeatureId = Features['featureId'];
+export type FeatureId = Features['featureId'] | 'OCS' | 'CNV';
```

Comment:
```suggestion
// TODO(jkilzi): remove 'OCS' once https://issues.redhat.com/browse/MGMT-7229 is completed
export type FeatureId = Features['featureId'] | 'OCS' | 'ODF' | 'CNV';
```

Reply:
since 'OCS' feature id isn't affected by the backend I'll change it to 'ODF' and not add both